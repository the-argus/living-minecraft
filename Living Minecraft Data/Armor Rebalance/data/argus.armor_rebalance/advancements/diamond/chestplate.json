{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:diamond_chestplate"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:diamond/chestplate",
    "loot": [
      "argus.armor_rebalance:diamond/chestplate"
    ]
  }
}
