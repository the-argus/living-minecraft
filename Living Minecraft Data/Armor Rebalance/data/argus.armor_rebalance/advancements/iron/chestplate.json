{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:iron_chestplate"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/iron/chestplate"
  }
}
