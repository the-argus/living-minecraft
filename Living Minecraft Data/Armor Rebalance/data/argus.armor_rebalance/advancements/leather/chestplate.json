{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:leather_chestplate"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:leather/chestplate",
    "loot": [
      "argus.armor_rebalance:leather/chestplate"
    ]
  }
}
