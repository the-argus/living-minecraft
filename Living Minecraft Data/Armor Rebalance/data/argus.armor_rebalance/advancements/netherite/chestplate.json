{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "argus.armor_rebalance:netherite_chestplate_shapeless"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/netherite/chestplate"
  }
}
