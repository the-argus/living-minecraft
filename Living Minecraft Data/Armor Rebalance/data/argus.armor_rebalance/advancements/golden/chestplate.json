{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:golden_chestplate"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:golden/chestplate",
    "loot": [
      "argus.armor_rebalance:golden/chestplate"
    ]
  }
}
